src/bp/ui-studio/src/web/components/Layout/StatusBar/index.jsx

```diff
@@ -146,6 +162,16 @@ class StatusBar extends React.Component {
           <ActionItem title="Notification" description="View Notifications" className={style.right}>
             <NotificationHub />
           </ActionItem>
+          <ActionItem
+            title="NLU Performance Status"
+            description={
+              this.state.nluStatus.f1score ? `f1: ${this.state.nluStatus.f1score}` : 'currently no f1 to display'
+            }
+            notclickable={this.state.nluStatus.computing.toString()}
```

Comment:
I see a potential trainrack here + a negation in the prop naming . you might want to use `disabled` its quite widely used